### PR TITLE
Feature/sscs 2293 terms cond page

### PR DESCRIPTION
**Pull request by iguannago**

### CODE REVIEW

File: terms-and-conditions-page/template.html [hunk 1]
```diff
@@ -0,0 +1,80 @@
+{% extends "look-and-feel/layouts/two_thirds.html" %}
+{% from "look-and-feel/components/phase_banner.njk" import phaseBanner %}
+
+{% block head -%}
+    <link href="{{ asset_path }}main.css" media="screen" rel="stylesheet" />
+{% endblock %}
+
+{% set phase="ALPHA" %}
```
Comment (bendiggle) [inline terms-and-conditions-page/template.html]: If you rebase master then you can remove here until line 14 and the beta banner should still appear

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js [hunk 2]
```diff
@@ -0,0 +1,112 @@
+'use strict';
+
+const paths = require('paths');
+const termsAndConditionsContent = require('terms-and-conditions-page/content.en.json');
+
+Feature('Terms and Conditions Page');
```
Comment (bendiggle) [inline test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: Do we really want to test all the content on the page? Maybe just do `I.see(pageTitle)` and then test all the links go to the correct urls? 
@paulgain

File: terms-and-conditions-page/template.html (hunk 1 above)
Comment (iguannago) [reply terms-and-conditions-page/template.html]: okey dokey. DONE

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 2 above)
Comment (iguannago) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: OK. Whatever you think guys. In my opinion, I agree, maybe it's too much, but tbh, it's part of the ticket requirements, on the other hand, and it's not going to harm if we leave it.  But, as I said, I am happy to follow whatever you decide guys. @paulgain , your input?

File: terms-and-conditions-page/content.en.json [hunk 3]
```diff
@@ -0,0 +1,80 @@
+{
+  "title": "Terms and conditions",
+  "termsAndConditions": {
+    "termsOfuse": "By using this service you’re agreeing to the following terms of use.",
```
Comment (paulgain) [inline terms-and-conditions-page/content.en.json]: Camel case please e.g. `termsOfUse`

Comment (iguannago) [reply terms-and-conditions-page/content.en.json]: Gosh...what an eye you have!

File: terms-and-conditions-page/content.en.json [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+  "relatedLinks": {
+    "title": "Related links",
+    "links": {
+      "cookie": {
+        "text": "Cookie policy",
+        "url": "/cookie-policy"
+      },
+      "privacy": {
+        "text": "GOV.UK privacy policy",
+        "url":"https://www.gov.uk/help/privacy-policy"
```
Comment (paulgain) [inline terms-and-conditions-page/content.en.json]: It's not a good idea to add URLs into a content file, where that file will be translated into another language. Ideally we should create a urls.js file defining every external URL we link to. This makes it far easier to maintain and update. TYA currently has one: https://git.reform.hmcts.net/sscs/track-your-appeal-frontend/blob/master/app/urls.js

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 2 above)
Comment (paulgain) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: I agree, that's a complete waste of time. If we did that for every page the end-to-end tests would take a significant amount of time to run.

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+
```
Comment (paulgain) [inline test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: Please remove all tabs, as previously mentioned we only do spaces. Thanks.
no-tabs-please

Comment (paulgain) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: Tabs show up as lines in Webstorm, spaces show up as dots. This is why the indentation is out on the PR.

File: terms-and-conditions-page/content.en.json (hunk 4 above)
Comment (paulgain) [reply terms-and-conditions-page/content.en.json]: I'll create a separate ticket for this as it appears the landing pages have the same problem, we can address them all together.

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 2 above)
Comment (iguannago) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: end to end tests will take a significant amount of time, eventually, anyway. And then we will have to define our smoke-tests and run that shorter tests often instead. Having said that, I am happy to remove all that if you guys think so :)

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 5 above)
Comment (bendiggle) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: *cough* IntelliJ *cough* @iguannago

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 2 above)
Comment (paulgain) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: Yep, I agree the tests take a significant amount of time to run, lets not make the situation worse.

Comment (iguannago) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: okey dokey :)

File: terms-and-conditions-page/content.en.json (hunk 4 above)
Comment (iguannago) [reply terms-and-conditions-page/content.en.json]: ticket is -> https://tools.hmcts.net/jira/browse/SSCS-2370

File: test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js (hunk 5 above)
Comment (iguannago) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: you guys finally made me move to Webstorm!

Comment (paulgain) [reply test/e2e/scenarios/terms-and-conditions-page/termAndConditionPage.js]: You know it makes sense.

**Pull request merged**